### ISSUE TITLE
Show weapon experience while attacking.

### ISSUE THREAD

**Issue by Mephiles** [Mephiles/torntools_2#201]
- Display weapon experience % when attacking someone.

**Comment by Harpush** [comment]
I think it should cover both weapon exp and finishing hits. Aimed for merits hunting too.

**Comment by DeKleineKobini** [comment]
Imo, this should be done before attacking

**Comment by Mephiles** [comment]
There's not really a good place to do it before attack, the thought here is that you probably have some weapon equipped that u want to get XP on but you might forget about it while attacking so showing the % in the attack is a good reminder.

**Comment by Harpush** [comment]
> There's not really a good place to do it before attack, the thought here is that you probably have some weapon equipped that u want to get XP on but you might forget about it while attacking so showing the % in the attack is a good reminder.

Ok true... But we do need to show weapon exp and finishing hit progression in items oage imo (but that's another issue)

**Comment by DeKleineKobini** [comment]
I'm looking at it right now, but I have absolutely no idea where to actually show it.
